wrt/wrt-uxmanu-android-tests/wrt-ux-app/testscripts/Crosswalk_WebApp_HTML5_Notification.feature

```diff
@@ -0,0 +1,7 @@
+Feature: wrt-ux-app
+ Scenario: Crosswalk WebApp HTML5 Notification
+  When launch "web_feature_notification_tests"
+    And I click button "Get Notification"
+    And I open notification
+   Then I should see text "New Email Received"
```

Comment:
I just assume that if this step fails, the notification will be always opened. Thus it will affect all the rest tests.